packages/server-core/src/githubapp/services.ts

```diff
@@ -0,0 +1,3 @@
+import GithubApp from './githubapp/githubapp.service'
```

Comment:
rather than creating a new `githubapp` folder lets move the sevice and class to `/server-core/src/projects/githubapp/...` instead

Reply:
Yes. got it!